x-pack/plugins/infra/public/pages/metrics/hosts/index.tsx

```diff
@@ -24,10 +25,24 @@ import { HostContainer } from './components/hosts_container';
 import { ExperimentalBadge } from '../../../components/experimental_badge';
 import { NoRemoteCluster } from '../../../components/empty_states';
 
-const HOSTS_FEEDBACK_LINK = 'https://ela.st/host-feedback';
+const HOSTS_FEEDBACK_LINK =
+  ' https://docs.google.com/forms/d/e/1FAIpQLScRHG8TIVb1Oq8ZhD4aks3P1TmgiM58TY123QpDCcBz83YC6w/viewform';
```

Comment:
One probably stupid question: Why do we need a space in front of the URL? Should this work?
```suggestion
  'https://docs.google.com/forms/d/e/1FAIpQLScRHG8TIVb1Oq8ZhD4aks3P1TmgiM58TY123QpDCcBz83YC6w/viewform';
```